app/src/pages/inside/launchesPage/modals/launchAnalysisModal/launchAnalysisModal.jsx

```diff
@@ -0,0 +1,237 @@
+import { InputRadio } from 'components/inputs/inputRadio';
+import { InputCheckbox } from 'components/inputs/inputCheckbox';
+import { LAUNCH_ANALYZE_TYPES } from 'common/constants/launchAnalyzeTypes';
+import { activeProjectSelector } from 'controllers/user';
+import classNames from 'classnames/bind';
+import styles from './launchAnalysisModal.scss';
+
+const cx = classNames.bind(styles);
+
+const messages = defineMessages({
+  analyseButton: {
+    id: 'analysisItemsModal.analyse',
+    defaultMessage: 'Analyse',
+  },
+  MOD_TITLE: {
+    id: 'analysisItemsModal.modTitle',
+    defaultMessage: 'Choose the base on which the Auto Analysis will be performed:',
+  },
+  OPTIONS_TITLE: {
+    id: 'analysisItemsModal.optionsTitle',
+    defaultMessage: 'Choose the test items that should be analyzed:',
+  },
+  ALL: {
+    id: 'analysisItemsModal.baseOptions.allLaunches',
+    defaultMessage: 'All launches',
+  },
+  LAUNCH_NAME: {
+    id: 'analysisItemsModal.baseOptions.withSameName',
+    defaultMessage: 'Launches with the same name',
+  },
+  CURRENT_LAUNCH: {
+    id: 'analysisItemsModal.baseOptions.current',
+    defaultMessage: 'Only current launch',
+  },
+  TO_INVESTIGATE: {
+    id: 'analysisItemsModal.itemOptions.investigate',
+    defaultMessage: 'To investigated items',
+  },
+  AUTO_ANALYZED: {
+    id: 'analysisItemsModal.itemOptions.byAA',
+    defaultMessage: 'Items analyzed automatically (by AA)',
+  },
+  MANUALLY_ANALYZED: {
+    id: 'analysisItemsModal.itemOptions.manually',
+    defaultMessage: 'Items analyzed manually',
+  },
+  VALIDATION_MESSAGE_CHOOSE_OPTION: {
+    id: 'analysisItemsModal.validation.chooseOption',
+    defaultMessage: 'You can not perform this operation unless at least one item is chosen',
+  },
+  VALIDATION_MESSAGE_CURRENT_LAUNCH: {
+    id: 'analysisItemsModal.validation.currentLaunch',
+    defaultMessage:
+      'You can not perform this operation for Auto-analyzed and Manually analyzed items simultaneously. Please choose one of them',
+  },
+  SUCCESS_MESSAGE: {
+    id: 'analysisItemsModal.successMessage',
+    defaultMessage: 'Auto-analyzer has been started.',
+  },
+});
+
+@withModal('analysisLaunchModal')
+@injectIntl
+@connect((state) => ({
+  activeProject: activeProjectSelector(state),
+}))
+@track()
+export class LaunchAnalysisModal extends Component {
+  static propTypes = {
+    intl: intlShape.isRequired,
+    data: PropTypes.shape({
+      item: PropTypes.object.isRequired,
+      onConfirm: PropTypes.func.isRequired,
+    }),
+    activeProject: PropTypes.string.isRequired,
+    tracking: PropTypes.shape({
+      trackEvent: PropTypes.func,
+      getTrackingData: PropTypes.func,
+    }).isRequired,
+  };
+
+  static defaultProps = {
+    data: {},
+  };
+  state = {
+    ANALYZER_MODE: LAUNCH_ANALYZE_TYPES.ANALYZER_MODE.LAUNCH_NAME,
+    ANALYZE_ITEMS_MODE: [LAUNCH_ANALYZE_TYPES.ANALYZE_ITEMS_MODE.TO_INVESTIGATE],
```

Comment:
We use such notation for constant variables.
There are not constants, please, use `analyzeItemsMode` and `analyzerMode`